pkg/controllers/migration/crd/controller.go

```diff
@@ -99,6 +100,9 @@ func (c *Controller) Reconcile(ctx context.Context, crd *apiextensionsv1.CustomR
 	stored := crd.DeepCopy()
 	crd.Status.StoredVersions = []string{"v1"}
 	if err := c.kubeClient.Status().Patch(ctx, crd, client.StrategicMergeFrom(stored, client.MergeFromWithOptimisticLock{})); err != nil {
+		if errors.IsConflict(err) {
```

Comment:
`client.IgnoreNotFound` here as well? I can't imagine how that would happen but probably doesn't hurt

Reply:
Added to `:102`